packages/docs/.gitignore

```diff
@@ -17,6 +17,7 @@ data
 
 # Production
 /build
+/docs
```

Comment:
This is not correct. `docs` needs to be there, `docs/api` should be ignored as the latter is autogenerated.